app/.eslintrc.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
```

Comment:
This is a duplicate and old file. We should delete it in favor of `.eslintrc.json`.